src/lib.rs

```diff
@@ -176,6 +176,42 @@ pub struct AvailableBalance {
     pub amount: Decimal,
 }
 
+/// Information to account owner can sometimes contain it's own structured data.
+/// It's difficult to find documentation for it, so the implementation should aim to be mostly
+/// consistent, while never dropping information from the field
+#[derive(Debug, Eq, PartialEq, Serialize, Deserialize, Clone)]
+#[serde(untagged)]
+pub enum InformationToAccountOwner {
+    Plain(String),
+    Structured {
+        transaction_code: String,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        posting_text: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        prima_nota: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        unknown11: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        purpose: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        applicant_bin: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        applicant_iban: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        applicant_name: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        return_debit_notes: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        recipient_name: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        unknown38: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        additional_purpose: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        unknown70: Option<String>,
```

Comment:
This verbosity could be avoided with [`skip_serializing_none`](https://docs.rs/serde_with/3.14.1/serde_with/attr.skip_serializing_none.html) but that would add a dependency.

Reply:
Let's go for the dependency. I think serde_with is pretty cool!